examples/golang/README.md

```diff
@@ -0,0 +1,42 @@
+# Golang Examples
+
+This folder contains examples that can be run with [wasmcloud][wasmcloud] which are written in [Golang][golang].
+
+Most examples compile with [TinyGo][tinygo] rather than the Golang compiler proper given that WebAssembly support landed in TinyGo first and is still slightly better supported there.
+
+## WebAssembly support
+
+As WebAssembly is intended to be a "compilation target" for native Golang code, upstream work is underway to integrate and improve support for the various standards of WebAssembly.
+
+| Language | Core Modules (`wasm32-unknown-unknown`) | Preview 1 (`wasm32-wasi-preview1`) | WASI Preview 2 (`wasm32-wasi-preview2`)  |
+|----------|-----------------------------------------|------------------------------------|------------------------------------------|
+| Golang   | ✅ (`GOOS=js`,`GOARCH=wasm`)            | ✅ (`GOOS=wasip1`)                 | ✅ (requires [adapter][wasi-p2-adapter]) |
+| TinyGo   | ✅ (`-target=wasm`)                     | ✅ (`-target=wasi`)                | ✅ (requires [adapter][wasi-p2-adapter]) |
+
+Tiny Go WASI support is detailed [on their website][tinygo-wasi].
+Golang WASI preview 1 support was [announced on the Golang blog][golang-blog-wasi]
+
+> [!INFO]
```

Comment:
This INFO also didn't show up with the correct formatting 🤔

Reply:
Same with the other READMEs, I'm not super familiar with why

Reply:
Oh this is weird -- that should be the right syntax for Github's alert...

Reply:
Ahh fixed, it was `[!NOTE]`